tests/data/state_estimation/current-sensor/basic-global-current-sensor/sym_output.json

```diff
@@ -0,0 +1,75 @@
+{
+  "version": "1.0",
+  "type": "sym_output",
+  "is_batch": false,
+  "attributes": {},
+  "data": {
+    "node": [
+      {
+        "id": 1,
+        "energized": 1,
+        "u_pu": 0.9999918620376325,
+        "u": 9999.918620376324,
+        "u_angle": -3.879563344990082e-12
+      },
+      {
+        "id": 2,
+        "energized": 1,
+        "u_pu": 0.9999924807982697,
+        "u": 9999.924807982698,
+        "u_angle": 4.379041916665773e-7
+      }
+    ],
+    "line": [
+      {
+        "id": 3,
+        "energized": 1,
+        "p_from": -165467.7877903657,
+        "q_from": 51153.76930561039,
+        "i_from": 9.999464137684235,
+        "s_from": 173194.3898884228,
+        "p_to": 165467.9125972454,
+        "q_to": -51185.14392598212,
+        "i_to": 10.0000000005913,
+        "s_to": 173203.7783485482
+      }
+    ],
+    "source": [
+      {
+        "id": 4,
+        "energized": 1,
+        "p": -165467.7877903657,
+        "q": 51153.76930561039,
+        "i": 9.999464137684235,
+        "s": 173194.3898884228,
+        "pf": -0.955387684482104
+      }
+    ],
+    "sym_load": [
+      {
+        "id": 5,
+        "energized": 1,
+        "p": -165467.9125972454,
+        "q": 51185.14392598212,
+        "i": 10.0000000005913,
+        "s": 173203.7783485482,
+        "pf": -0.955336618511867
+      }
+    ],
+    "sym_voltage_sensor": [
+      {
+        "id": 6,
+        "u_residual": 0.08137962367538165,
+        "u_angle_residual": -1.22213342841859e-13
```

Comment:
why not just make it `0` by means of `rtol` and `atol`? or is this the exact solution?